website/src/js/components/SearchOptionsDropdown.js

```diff
@@ -43,7 +43,7 @@ class SearchOptionsDropdown extends Component {
 
     this.props.trackEvent({
       category: 'dropdown',
-      action: 'toggle | Advanced',
+      action: 'toggle | Search Options',
```

Comment:
I know this is a GA event and you'll probably lose historical data but I it's probably good to update this too.